src/bin/rav1e.rs

```diff
@@ -34,12 +35,13 @@ fn main() {
     framerate.den
   );
 
-  let mut last_rec: Option<Frame> = None;
+  let mut rec_buffer: [Option<Rc<Frame>>; 8] = [None, None, None, None, None, None, None, None];
   loop {
     //fi.frame_type = FrameType::KEY;
     fi.frame_type =
       if fi.number % 30 == 0 { FrameType::KEY } else { FrameType::INTER };
 
+    fi.refresh_frame_flags = if fi.frame_type == FrameType::KEY { 0xff } else { 1 };
```

Comment:
We have "const REF_FRAMES: u32 = 8;" in src/lib.rc,
Then,
let allFrames = (1 << REF_FRAMES) - 1;
Can we replace '0xff' with allFrames? (Sorry..... *_*)

Reply:
Though the av1 spec used named constants, I think we we can keep as it is since we all knows what 8 is for.
For ex, when av1 spec calculates internal frame width and height, it uses numbers directly instead of named constants. So, pls skip my comments above.